pkg/config/validation/validation.go

```diff
@@ -3187,6 +3183,12 @@ func validateLocalityLbSetting(lb *networking.LocalityLoadBalancerSetting) error
 	}
 
 	for _, failover := range lb.GetFailover() {
+		if failover.From == "" || failover.To == "" {
+			return fmt.Errorf("locality lb failover settings must specify both From and To")
+		}
+		if len(strings.SplitN(failover.From, "/", 3)) > 1 || len(strings.SplitN(failover.To, "/", 3)) > 1 {
+			return fmt.Errorf("locality lb failover settings only different regions can be set and should not be start or end with '/'")
```

Comment:
locality lb failover settings must not contain `/`

Reply:
done